lang/py/src/avro/schema.py

```diff
@@ -43,9 +43,17 @@
 import json
 import math
 import sys
+from warnings import warn
```

Comment:
Not a huge deal, but I’ve been slowly trying to iterate avro to import modules over names. First, it gives a touch of clear provenance in the namespace of a name. Second, it enables module reloading during debugging.